src/main/java/it/eng/idsa/businesslogic/processor/sender/SenderUsageControlProcessor.java

```diff
@@ -79,7 +79,6 @@ public void process(Exchange exchange) {
 
 		logger.info("Proceeding with Usage control enforcement");
 		logger.info("from: " + exchange.getFromEndpoint());
```

Comment:
Do we need this fromEndpoint log info?

Reply:
Since it's already defined for the consumer side I agree with you that it is not needed